datadog_checks_dev/tests/plugin/test_environment_runner.py

```diff
@@ -1,12 +1,16 @@
 # (C) Datadog, Inc. 2018
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
+import pytest
 import json
 from base64 import urlsafe_b64decode
 
 
 def test_runner(dd_environment_runner, mock_e2e_config, mock_e2e_metadata):
     message = dd_environment_runner
+    if not message:
+        # this can happen when tests are manually run
+        pytest.skip("ddev plugin not loaded, skipping plugin test...")
```

Comment:
This is not good (means no aggregator too) and should never happen when deps are present. How did this happen?

Reply:
Using `pytest` directly

Reply:
I don't get how this would happen still, since the plugin is a `pytest` plugin

Reply:
Plugin is only enabled when `DDEV_TESTING_PLUGIN` env var is set to `true`. I've changed the skip logic so it's more explicit.